DSharpPlus/DebugLogger.cs

```diff
@@ -59,33 +61,35 @@ public void LogTaskFault(Task task, LogLevel level, string application, string m
 
         internal void LogHandler(object sender, DebugLogMessageEventArgs e)
         {
-            switch (e.Level)
+            lock (_lock)
             {
-                case LogLevel.Debug:
-                    Console.ForegroundColor = ConsoleColor.DarkGreen;
-                    break;
-
-                case LogLevel.Info:
-                    Console.ForegroundColor = ConsoleColor.White;
-                    break;
-
-                case LogLevel.Warning:
-                    Console.ForegroundColor = ConsoleColor.DarkYellow;
-                    break;
-
-                case LogLevel.Error:
-                    Console.ForegroundColor = ConsoleColor.DarkRed;
-                    break;
-
-                case LogLevel.Critical:
-                    Console.BackgroundColor = ConsoleColor.DarkRed;
-                    Console.ForegroundColor = ConsoleColor.Black;
-                    break;
+                switch (e.Level) {
```

Comment:
We don't use Java braces here.

Reply:
God damn, I know, I saw, I thought I changed that...
Changed it.